casbin/model/scope_config.cpp

```diff
@@ -278,6 +278,23 @@ void EvalNoResult(Scope scope, std::string expression){
     duk_eval_string_noresult(scope, expression.c_str());
 }
 
+void DeletePropFromObject(Scope scope, std::string object_name, std::string prop_name) {
+    if (duk_get_global_string(scope, object_name.c_str())) {
+        duk_del_prop_string(scope, -1, prop_name.c_str());
+    }
+}
+
+void CleanScope(Scope scope) {
+    std::vector<std::string> props = {"obj", "sub", "act"};
+    std::vector<std::string> objects = {"r", "p"};
```

Comment:
Again, multiple definitions can be accommodated in a single config. eg: r1, r2, r3, p1, p2, p3 etc.